docs/source/asr/models.rst

```diff
@@ -128,6 +128,16 @@ You may find the example config files of Conformer-Transducer model with charact
 ``<NeMo_git_root>/examples/asr/conf/conformer/conformer_transducer_char.yaml`` and
 with sub-word encoding at ``<NeMo_git_root>/examples/asr/conf/conformer/conformer_transducer_bpe.yaml``.
 
+Fast-Conformer
+--------------
+
+The Fast Conformer (CTC and RNNT) models have a faster version of the Conformer encoder and differ from it as follows:
```

Comment:
Mention that it's a subword based ctc model (8x + char doesn't work unless it's RNNT). RNNT can support char and subword

Reply:
done